metasv/process_age_alignment.py

```diff
@@ -114,7 +113,8 @@ def overlap(s1, e1, s2, e2):
 
 
 def select_best_assembly(assemblies):
-    if not assemblies: return age.AgeRecord()  # return dummy assembly
+    if not assemblies:
+        return age.AgeRecord()  # return dummy assembly, Marghoob this is not defined?
```

Comment:
the keyword argument age_out_file is None by default if not specified. so, age.AgeRecord would work just fine. and the module "age" is visible to this file since it is in the same directory.